src/Controller/Traits/RequestTrait.php

```diff
@@ -36,18 +37,22 @@ trait RequestTrait
      */
     protected string $expectedStateAuthId = OAuth2::AUTHID;
 
+    /** @var array|null  */
+    private ?array $requestParams;
```

Comment:
I don't see anything that sets this object field. Only something that reads it.

Reply:
I updated the code and it is now used as it should be.